features/testdebt_generic_test_data_step.feature

```diff
@@ -0,0 +1,31 @@
+@deed_overrides @wip
+Feature: Override deed variables
+
+Acceptance Criteria
+- Can override ALL deed variables from one step
+
+@techdebt @techdebt1
+Scenario: WIP for default deed
+Given I create default deed with <2> borrowers
+And borrower <2> views the deed
+And borrower <1> views the deed
+
+# version 1.6
+# Readable way to override deed info
+@techdebt @techdebt2
+Scenario: WIP for override
+Given I setup deed with <1> borrowers
+Then set "forename" to "Timmy" for borrower <1>
+And I create the deed
+And borrower <1> views the deed
+And the borrower <1> signature element is present on page
+Then the deed is digitally signed by borrower <1>
```

Comment:
Definitely prefer the method above. Though it does involve three steps instead of one, it definitely reads smoother, and consequently I would imagine it's easier to understand what the test is doing and what code is hidden behind those steps. Plus it lets us be specific about which borrower we want to amend which will make this more reusable. Only thing I would say if there is space is to make the steps a bit more englishy, for example Given I setup **a** deed with <1> borrowers and Then **I amend** "" to "" for borrower <>. Just an example but really minor so don't worry if it breaks the line length doing this.